src/components/popups/Pokemon.jsx

```diff
@@ -427,12 +427,12 @@ const ExtraInfo = ({ pokemon, perms, t }) => {
       ))}
       {[first_seen_timestamp, updated].map((time, i) => (
         <Fragment key={time}>
-          <Grid item xs={5} style={{ textAlign: 'center' }}>
+          <Grid item xs={7} style={{ textAlign: 'center' }}>
             <Typography variant="caption" align="center">
               {i ? t('lastSeen') : t('firstSeen')}:
             </Typography>
           </Grid>
-          <Grid item xs={6} style={{ textAlign: 'right' }}>
+          <Grid item xs={4} style={{ textAlign: 'right' }}>
```

Comment:
![Screen Shot 2021-08-06 at 12 45 50 PM](https://user-images.githubusercontent.com/58572875/128544425-f1293aab-88db-47c2-807e-3138883f4155.png)

The pokestop and gyms are fine but this one causes some issues for English.

Reply:
Gotcha my mistake. I will remove changes related to styling for now